source/manuals/programming-languages/python/python.html.md.erb

```diff
@@ -0,0 +1,94 @@
+---
+title: Writing Python at GDS
+expires: living
+---
+
+# <%= current_page.data.title %>
+
+
+### About this manual:
+
+This manual is designed to aid developers in writing Python code that is clear and consistent, within, and across,
+projects at GDS.
+
+The [Google Python style guide][GPSG] outlines sensible conventions for
+writing clear and concise Python code and we would suggest using this as a reference unless something is explicitly
+mentioned in the [GDS Python Style Guide][GDSPSG].
+
+
+### IDEs
+
+[JetBrains PyCharm][PyCharm] is commonly used at GDS but IDE choice is not enforced as part of this guide.
```

Comment:
So this was inspired by the Java styleguide PR:
https://github.com/alphagov/gds-tech/pull/43/files#diff-f687851c0d55f2a553ca4922cf36dc5aR8

It's mainly because I'd strongly advise people new to Python looking for an IDE to use PyCharm. It's the right tool for the job out of the box.

Could change the wording to make it clear that it's a recommendation?

Reply:
For Java, all of our Java teams use IntelliJ IDEA already. Is it the case that all of our Python teams use PyCharm?

Reply:
Is that just Notify and Marketplace?

Reply:
Possibly some performance platform.

Reply:
I guess what I need to know is: what's the precursor for keeping this in? 100% of people use it? because it's definitely not 100%. 50%? Maybe. A person on each Python project? Almost certainly.

Reply:
Without this comment about which IDE's are used I'd expect people will do as they do now, using whichever editor/IDE they are most familiar with. As far as I'm aware we don't have any desire to change that. 

There are cases for sharing tooling e.g. a team may make a decision to install PyCharm across all developer laptops make pairing easier, but I don't see a need to mention that in the GDS way.

Writing Java requires tooling in a way unlike Python and benefits more from sharing that tooling with others on your team. For example it's common to have your [build system generate project files specific to IDEs](https://github.com/alphagov/verify-service-provider/blob/master/build.gradle#L3) supporting many IDE's is challenging as not everyone is equally effected by IDE integration with the project (e.g. code layout, build scripts and automation).

Reply:
OK, looks like I got the wrong end of the stick as to why the Java PR needed an IDE section. I've removed the Python IDE section altogether but let me know if you think it's worth saying 'pick one with Python support' and I'll put this back in:

> Whatever your choice something with decent Python support is going to make complying with the suggestions in this guide far easier. If you haven't worked with Python before then your first port of call should be the people you're working with. This will enable you to get help quickly when you need it.